src/Application/Logger/Logger.ts

```diff
@@ -13,90 +13,45 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import pino from 'pino';
+import { configure, getLogger, shutdown } from 'log4js';
 import { homedir } from 'os';
 import { existsSync, mkdirSync } from 'fs';
 import { join } from 'path';
 
+export const DEBUG = 'debug';
+export const ERROR = 'error';
+
 const logPath = join(homedir(), '.ossindex');
 
-const logPathFile = process.env.CI ? undefined : join(logPath, '.auditjs.combined.log');
+const logPathFile = join(logPath, '.auditjs.combined.log');
```

Comment:
Just confirming we're doing 1 combined log file now

Reply:
Yep!